precommit.mk

```diff
@@ -0,0 +1,10 @@
+ifneq (, $(shell which pre-commit))
```

Comment:
This is so we don't break on environments that don't have `pre-commit` installed (e.g. within docker).